src/services/utilities.ts

```diff
@@ -2462,7 +2462,7 @@ export function createModuleSpecifierResolutionHost(program: Program, host: Lang
     return {
         fileExists: fileName => program.fileExists(fileName),
         getCurrentDirectory: () => host.getCurrentDirectory(),
-        readFile: maybeBind(host, host.readFile),
+        readFile: host.readFile.bind(host),

```

Comment:
_Maybe_ this should be:

```suggestion
        readFile: fileName => host.readFile(fileName),
```

Just to match the rest of these?

Or maybe we should have `maybeBind` overload like:

```ts
/** @internal */
export function maybeBind<T, A extends any[], R>(obj: T, fn: (this: T, ...args: A) => R): (...args: A) => R;
/** @internal */
export function maybeBind<T, A extends any[], R>(obj: T, fn: ((this: T, ...args: A) => R) | undefined): ((...args: A) => R) | undefined;
```

Mainly, using `bind` directly is dubious just because there's no type safety.

Reply:
Changed it to use a arrow function.